torchbench.py

```diff
@@ -115,22 +119,47 @@
 
 # These benchmarks took >60s on an i9-11900K CPU
 SLOW_BENCHMARKS = {
-    *{
-        "BERT_pytorch",  # 137s
-        "demucs",  # 116s
-        "fastNLP_Bert",  # 242s
-        "hf_Albert",  # 221s
-        "hf_Bart",  # 400s
-        "hf_Bert",  # 334s
-        "hf_DistilBert",  # 187s
-        "hf_GPT2",  # 470s
-        "hf_Reformer",  # 141s
-        "speech_transformer",  # 317s
-        "vision_maskrcnn",  # 99s
-    },
     *VERY_SLOW_BENCHMARKS,
+    "BERT_pytorch",  # 137s
```

Comment:
What are these times measuring? how long they take to run?

Reply:
Yes correct.

Reply:
This implements `--fast` which skips slow CPU benchmarks.  (A minor quality of life feature for correctness testing.)